File: frontend/lib/models/report.dart
class Report {
  final int id;
  final int reportNumber;
  final int departmentId;
  final int jobId;
  final int employeeId;
  final int accountId;

  // Новые поля
  final String dateReceived;
  final String amountIssued;
  final String dateApproved;
  final String purpose;
  final String recognizedAmount;
  final String comments;

  Report({
    required this.id,
    required this.reportNumber,
    required this.departmentId,
    required this.jobId,
    required this.employeeId,
    required this.accountId,
    required this.dateReceived,
    required this.amountIssued,
    required this.dateApproved,
    required this.purpose,
    required this.recognizedAmount,
    required this.comments,
  });

  factory Report.fromJson(Map<String, dynamic> json) {
    return Report(
      id: json['id'],
      reportNumber: json['reportNumber'],
      departmentId: json['departmentId'],
      jobId: json['jobId'],
      employeeId: json['employeeId'],
      accountId: json['accountId'],
      dateReceived: json['dateReceived'],
      amountIssued: json['amountIssued'],
      dateApproved: json['dateApproved'],
      purpose: json['purpose'],
      recognizedAmount: json['recognizedAmount'],
      comments: json['comments'],
    );
  }
}

File: monolith/src/main/java/com/example/monolith/report/Report.java
package com.example.monolith.report;

import com.example.monolith.account.Account;
import com.example.monolith.department.Department;
import com.example.monolith.employee.Employee;
import com.example.monolith.job.Job;
import jakarta.persistence.*;
import lombok.Getter;
import lombok.RequiredArgsConstructor;
import lombok.Setter;

@Entity
@Getter
@Setter
@RequiredArgsConstructor
public class Report {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @ManyToOne
    private Department department;

    @ManyToOne
    private Job job;

    @ManyToOne
    private Employee employee;

    @ManyToOne
    private Account account;

    private int reportNumber;
    private Long departmentIdentifier;
    private Long jobIdentifier;
    private Long employeeIdentifier;
    private Long accountIdentifier;

    private String dateReceived;       // Дата получения д/с
    private String amountIssued;       // Выданная сумма
    private String dateApproved;       // Дата утверждения а/о
    private String purpose;            // Назначение
    private String recognizedAmount;   // Признанная сумма затрат по а/о
    private String comments;           // Комментарии
}
